packages/components/src/popover/index.js

```diff
@@ -195,41 +195,39 @@ const Popover = (
 	 * Store the offset in a ref, due to constraints with floating-ui:
 	 * https://floating-ui.com/docs/react-dom#variables-inside-middleware-functions.
 	 */
-	const frameOffset = useRef( getFrameOffset( ownerDocument ) );
```

Comment:
Renamed variable, removed initial value (it gets updated in the `useLayoutEffect` anyway)

Reply:
My only concern with removing it on the initial render is that we don't have control over how floating ui applies the offset. I felt it was safer to ensure it was defined from the start.

Reply:
Added back as part of https://github.com/WordPress/gutenberg/pull/43329/commits/4bf6dfc181e7b4b492b06504b5426d130e88c53f.

As I worked through this change, I realized that we also need to store the `offset` prop in a `ref` if we want the middleware to always use the latest value